site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/pfi/PfiQuotesTest.kt

```diff
@@ -0,0 +1,74 @@
+package website.tbd.developer.site.docs.tbdex
+
+import tbdex.sdk.protocol.models.Rfq
+import tbdex.sdk.protocol.models.Quote
+import de.fxlae.typeid.TypeId
+import tbdex.sdk.httpserver.models.*
+import java.util.NoSuchElementException
+import tbdex.sdk.protocol.models.*
+import tbdex.sdk.httpserver.TbdexHttpServer
+import tbdex.sdk.httpserver.TbdexHttpServerConfig
+import web5.sdk.crypto.AwsKeyManager
+import web5.sdk.dids.methods.dht.DidDht
+import tbdex.sdk.httpserver.models.SubmitKind
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.Did
+import web5.sdk.dids.methods.jwk.DidJwk
+import java.time.OffsetDateTime
+import website.tbd.developer.site.docs.tbdex.*
+
+class PfiQuotesTest {
+  
+  suspend fun createQuoteFromRfq(message: Message) {
+    val offeringsApiProvider = OfferingsApiProviderTest()
+    val exchangesApiProvider = ExchangesApiProviderTest()
+    val pfiDid = DidJwk.create(InMemoryKeyManager())
+    val dataProvider = MockDataProviderTest()
+
+    // :snippet-start: pfiQuotesWriteKt
+    val data = mapOf(
```

Comment:
to be in sync with JS example, add the preceding comment
```suggestion
   // Write the message to your exchanges database
    val data = mapOf(
```